.github/workflows/ci.yml

```diff
@@ -0,0 +1,97 @@
+name: Continuous integration
+
+on:
+  push:
+    branches: [develop, master]
+  pull_request:
+
+
+env:
+  SECRET_KEY: topsecret123
+
+
+jobs:
+  test:
+    name: Tests
+    runs-on: ubuntu-latest
+    container: python:3.7-buster
```

Comment:
If wanting to use container in top of VM, better to use same than app really uses `helsinkitest/python:3.7-slim`.

But services would work also directly to runner: https://github.com/actions/example-services/blob/main/.github/workflows/postgres-service.yml#L46-L72

Reply:
Changed to run directly on the runner.